files/entrypoint.sh

```diff
@@ -34,6 +34,9 @@ if [ -f /firstrun ]; then
 	# Configure WEB UI access
 	sed -ie "s/^\$Conf{CgiAdminUsers}\s*=\s*'\w*'/\$Conf{CgiAdminUsers} = '${BACKUPPC_WEB_USER:-backuppc}'/g" /etc/backuppc/config.pl
 	htpasswd -b -c /etc/backuppc/htpasswd ${BACKUPPC_WEB_USER:-backuppc} ${BACKUPPC_WEB_PASSWD:-password}
+	
+	# Workaround bug which causes BackupPC fail to start when container launched with existing configuration
+	sed -ie "s|^\$Conf{CgiURL}\s*=\s*''http://localhost/cgi-bin/BackupPC/BackupPC_Admin''|\$Conf{CgiURL} = 'http://localhost/cgi-bin/BackupPC/BackupPC_Admin'|g" /etc/backuppc/config.pl
```

Comment:
In general we would except `http://localhost/cgi-bin/BackupPC/BackupPC_Admin` to be used in the container context, as exotic values are likely to fail. However, for the sake of a complete coverage of the fix, could you use a regex to capture any content in the double quote for `$Conf{CgiURL}` and reinject it in single quote ?

Something like:
```
sed -ie "s|^\$Conf{CgiURL}\s*=\s*''\(.*\)''|\$Conf{CgiURL} = '\1'|g" /etc/backuppc/config.pl
```

Reply:
Sure, this can be done, but in BackupPC v3 "hostname" is always == "localhost"

https://github.com/adferrand/docker-backuppc/blob/fa81021cfc873df62f40ab38768e619454f7d7d3/files/entrypoint.sh#L29

So, I did a quick assumption here. ^__^

Reply:
Yes, on v4, the hostname value is configurable, but not on v3. Still, I prefer to be generic. I tested the sed command on my side, it works.

Reply:
Thank you for testing!